### ISSUE TITLE
Error running `rails db:gis:setup`

### ISSUE THREAD

**Issue by code-later** [rgeo/activerecord-postgis-adapter#328]
First of all I would like to thank you for your great work and this project.

With the current version I receive an error when running `rails db:gis:setup`:

```
$ rails db:gis:setup --trace                                                                                                                
** Invoke db:gis:setup (first_time)
** Invoke db:load_config (first_time)
** Invoke environment (first_time)
** Execute environment
** Execute db:load_config
** Execute db:gis:setup
rails aborted!
NoMethodError: undefined method `configuration_hash' for #<Hash:0x0000560ce59781f8>
/gems/activerecord-6.1.1/lib/active_record/tasks/postgresql_database_tasks.rb:21:in `initialize'
/gems/activerecord-postgis-adapter-7.0.0/lib/active_record/connection_adapters/postgis/postgis_database_tasks.rb:8:in `initialize'
/gems/activerecord-postgis-adapter-7.0.0/lib/active_record/connection_adapters/postgis/databases.rake:14:in `new'
/gems/activerecord-postgis-adapter-7.0.0/lib/active_record/connection_adapters/postgis/databases.rake:14:in `block (5 levels) in <main>'
/gems/activerecord-postgis-adapter-7.0.0/lib/active_record/connection_adapters/postgis/databases.rake:13:in `each'
/gems/activerecord-postgis-adapter-7.0.0/lib/active_record/connection_adapters/postgis/databases.rake:13:in `block (4 levels) in <main>'
/gems/activerecord-postgis-adapter-7.0.0/lib/active_record/connection_adapters/postgis/databases.rake:9:in `each'
/gems/activerecord-postgis-adapter-7.0.0/lib/active_record/connection_adapters/postgis/databases.rake:9:in `block (3 levels) in <main>'
```

It seems something was missed in #322. It seems the updated code expects the `env` itself and not the final configuration hash. When looking at the Rake-task (https://github.com/rgeo/activerecord-postgis-adapter/blob/master/lib/active_record/connection_adapters/postgis/databases.rake#L14) passing `env` instead of `env.config` fixes this issue.

At least it did for me locally.

Thanks for looking into this.

**Comment by keithdoggett** [comment]
Thanks for pointing this out! I'm seeing this on my end too. I'm going to try to make a failing test case to verify, but it looks like your fix should work.

**Comment by keithdoggett** [comment]
I opened #329. It's going to be difficult (if not impossible) to test this in the current suite because the task is dependent on rails. Perhaps in the future, we could set up another workflow that creates a rails app and runs the rake tasks to make sure this is properly tested.

**Comment by code-later** [comment]
Thanks @keithdoggett for this fast response and taking care of it. These things are always quite hard to catch.

**Comment by keithdoggett** [comment]
Released in 7.0.1
